README.md

```diff
@@ -49,7 +58,74 @@ The `fairlearn` package contains the following algorithms for mitigating unfairn
 > _Note_:
 > DP refers to demographic parity, EO to equalized odds, and BGL to bounded group loss. For more information on these and other terms we use in this repository please refer to the [terminology page](./TERMINOLOGY.md). To request additional algorithms or fairness definitions, please open a [new issue](https://github.com/fairlearn/fairlearn/issues).
 
-## Install fairlearn
+
+## Visualizations
+
+This repository containts a visualization dashboard that enables data scientists, auditors, business decision makers, and other stakeholders to visually observe and analyze how their model predictions are impacting different groups of data points (e.g., different ethnicities). It provides a high-level and detailed view of prediction insights for assessing which groups are negatively impacted. Using this dashboard you get insights into (un)fairness of you model and gain ideas around potential ways of mitigating the obsereved unfairness.  
+
+### Visualization: single model, pre-mitigation 
+
+Load the visualization dashboard in your noteboook to observe and assess your model's fairness using the following code:
```

Comment:
Since this isn't a tutorial, should probably start this 'To load' and change the rest of the sentence to flow from that.

Reply:
done.